pkg/smartcontract/manifest/standard/comply.go

```diff
@@ -66,6 +67,10 @@ func ComplyABI(m *manifest.Manifest, st *Standard) error {
 }
 
 func comply(m *manifest.Manifest, checkNames bool, st *Standard) error {
```

Comment:
seems like I have broken it.  because:
```
 ./bin/neo-go contract compile -i ./examples/nft-nd/nft.go -c ./examples/nft-nd/nft.yml -o examples/nft-nd/nft.nef -m ./examples/nft-nd/nft.manifest.json
error while trying to compile smart contract file: generics are currently unsupported, please, see the https://github.com/nspcc-dev/neo-go/issues/2376: time.parseRFC3339 has function type parameters

```

Reply:
It's because you have added standard go library packages to the smart contract. Our compiler is not so smart, it can't compile standard go libraries. Remove all mentions of `*big.Int` and etc. from the contract.